ql/ql/src/queries/style/ConsistentAlertMessage.ql

```diff
@@ -0,0 +1,77 @@
+/**
+ * @name Consistent alert message
+ * @description The alert message should be consistent across languages.
+ * @kind problem
+ * @problem.severity warning
+ * @id ql/consistent-alert-message
+ * @tags correctness
+ * @precision very-high
+ */
+
+import ql
+
+/**
+ * Gets a string representation of the entire message in `sel`.
+ * Ignores everything that is not a string constant.
+ */
+string getMessage(Select sel) {
+  result =
+    strictconcat(String e, Location l |
+      // is child of an expression in the select (in an uneven position, that's where the message is)
```

Comment:
>  ... in an uneven position ...

That's an _odd_ way to phrase that. :wink: 

(Though I'm not sure "odd position" is actually more intuitive. Maybe "odd-indexed position" would be better?)

Reply:
> Though I'm not sure "odd position" is actually more intuitive.

That's not an odd position to have 😉  

I'll go for `odd-indexed position`.